Formula/tinyscheme.rb

```diff
@@ -20,35 +20,38 @@ def install
     share.install("init.scm")
     bin.install("scheme")
   end
+
+  test do
+    assert_match "Usage: tinyscheme", shell_output("#{bin}/scheme -\?", 1)
+  end
 end
 
 __END__
---- a/makefile  2011-01-16 20:51:17.000000000 +1300
-+++ b/makefile  2012-04-08 22:38:11.000000000 +1200
+--- a/makefile
++++ b/makefile
```

Comment:
@DomT4 
Moving this to `homebrew/patches` sounds good to me. 

Is the standard operating procedure just to use the `raw.githubusercontent.com` URL?

@ilovezfs I'm hesitant about that approach, mainly because it'll most likely execute successfully but produce the wrong build specifications if the `tinyscheme` maintainers decide to tweak the makefile again. With the diff, the patch will fail early and we won't be stuck debugging compilation/behaviour errors.